soc/nordic/common/vpr/soc_init.c

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (C) 2024 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/init.h>
+#include <hal/nrf_vpr_csr.h>
+
+static int vpr_init(void)
+{
+	nrf_vpr_csr_rtperiph_enable_set(true);
+
+	return 0;
+}
+
+SYS_INIT(vpr_init, PRE_KERNEL_1, 0);
```

Comment:
Why not do this inside `mbox_nrf_vevif_local.c`?

Reply:
Because VEVIF is one of multiple RT peripherals (not in zephyr code yet), and the enable is shared between those.
So making this on SOC-level makes more sense than enabling it redundantly in each driver.